tangelo/linq/target/target_stim.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The stim simulation package is made for fast simulation of Clifford circuits.
+See https://arxiv.org/abs/2103.02202v3 for more details.
+"""
+
+from collections import Counter
+
+import numpy as np
+
+from tangelo.linq import Circuit
+from tangelo.linq.target.backend import Backend
+from tangelo.linq.translator import translate_circuit as translate_c
+from tangelo.linq.translator.translate_stim import translate_tableau
+
+
+class StimSimulator(Backend):
+    """Interface to the stim simulator"""
+    def __init__(self, n_shots=None, noise_model=None):
+        import stim
+        super().__init__(n_shots=n_shots, noise_model=noise_model)
+        self.stim = stim
+
+    def simulate_circuit(self, source_circuit: Circuit, return_statevector=False, initial_statevector=None):
+        """Perform state preparation corresponding to the input circuit on the
+        target backend, return the frequencies of the different observables, and
+        either the statevector or None depending on the availability of the
+        statevector and if return_statevector is set to True. For the
+        statevector backends supporting it, an initial statevector can be
+        provided to initialize the quantum state without simulating all the
+        equivalent gates.
+
+        Args:
+            source_circuit: a circuit in the abstract format to be translated
+                for the target backend.
+            return_statevector(bool): option to return the statevector as well,
+                if available.
```

Comment:
you tell me, it's your class. You `backend_info` function should say that.

Probably a vestige of when we only had one class